packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -391,7 +413,44 @@ class _MethodStreamManager {
     WebSocket webSocket,
     Session session,
     SerializationManager serializationManager,
+    _RequiredAuthentication? requiredAuthentication,
   ) {
+    late void Function(SerializableModel) authenticationRevokedCallback;
```

Comment:
Suggestion: Name with similar convention to the event that will invoke this callback. The event is `MessageCentralServerpodChannels.revokedAuthentication`, so I would suggest naming this `revokedAuthenticationCallback`.